client/src/varclient.c

```diff
@@ -176,8 +179,23 @@ int ClientRequest( VarClient *pVarClient, int signal )
             {
                 do
                 {
-                    pVarClient->blocked = 1;
-                    result = sem_wait( &pVarClient->sem );
+                    if ( pVarClient->requestTimeout_ms > 0) {
```

Comment:
sorry - coding standard on TGP code has compound statement opening parens on the next line

Reply:
fixed